docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -8,12 +8,12 @@ The Cassandra connector allows querying data stored in
 Requirements
 ------------
 
-Requirements for using the connector in a catalog to connect to a Cassandra
-data source are:
+To connect to a Cassandra data source using the connector in a catalog, you
```

Comment:
What about `Connecting to a <XYZ> data source using the connector requires:`?

`connector in a catalog` is a bit confusing (specially that the reader hasn't been yet introduced to catalog <-> connector relation. The catalog and connector relationship is better suited in the Configuration section IMO.

Reply:
We discussed this and want to leave it as is. The connection between catalog and connector is very important and should be here as well. A common misunderstanding is that you can configure a connector separately somehow, when in fact you can not ..